cli/README.md

```diff
@@ -65,6 +65,8 @@ Options:
   (use quotes to provide multiple args).
 - use `--inputFile <url...>` to provide one or more input files to your iApp
   during testing.
+- use `--requesterSecret <key=value...>` to provide one or more requester
+  secrets to your iApp during testing.
```

Comment:
What do you think of having here the same description as the one we have in the CLI itself? (the one added in `src/index.js`)

The two first options `--args` and `--inputFile` are kind of ok to understand, but `--requesterSecret` is more cryptic for a non-iExec developer. I wouldn't assume anyone getting to this README understand what it is.
 - Briefly explain it here?
 - Include link to https://protocol.docs.iex.ec/for-developers/technical-references/application-io#requester-secrets ?
 - Give a meaningful example: "For an iApp like web3mail, requester secrets include "emailSubject" or "senderName".

Reply:
added links to doc ddc80a06ac0ac30b6e46814259c81676bbca47fb